### PR TITLE
completed css-two-column-layout exercise

### PR DESCRIPTION
**Pull request by kvnliang4**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  width: auto;
+  max-height: auto;
+}
+
+.column-full {
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  width: 60%;
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css]: always make sure that the column size matches the column name, if you want to the content to be 60% of the column, you can have the column at 100% width for a column full, and then adjust the width of the child element.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking really good! Just a semantic update is needed in the CSS

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (kvnliang4) [reply css-two-column-layout/styles.css]: Thanks, the edit worked out nicely

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**